docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -94,10 +94,10 @@ Priority levels
 ------------------
 
 During load balancing, Envoy will generally only consider hosts configured at the highest priority
-level.  For each EDS ``LocalityLbEndpoint``, an optional priority may also be specified.  Currently
-the failover mechanics for routing from one priority level to another are fairly simplistic: a given
-priority level will be used until it has zero healthy hosts, at which ponint it will hard fail to
-the next highest priority level.
+level. For each EDS :ref:LocalityLbEndpoint <envoy_api_msg_LocalityLbEndpoint> an optional priority
```

Comment:
I don't think this is valid syntax; my example had back ticks eaten by GH MD, it shuold have been:

```
:ref:`LocalityEndpoint <stuff>`
```

Did you look at the generated docs?

Reply:
no, I mistakenly thought the build would catch syntax errors, but I think the lack of backticks caused it to not even check for me.
Also LbEndpoints, now that it's checking for me.